### PR TITLE
infra: Support webui-desktop script by makeupdates

### PR DESCRIPTION
**Pull request by jkonecny12**
This could help us with future debugging of the issues.

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Thanks!

File: scripts/makeupdates [hunk 1]
```diff
@@ -276,6 +276,8 @@ def copy_updated_files(tag, updates, cwd, builddir):
             install_to_dir(gitfile, "usr/sbin")
         elif any(gitfile.endswith(libexec_script) for libexec_script in LIBEXEC_SCRIPTS):
             install_to_dir(gitfile, "usr/libexec/anaconda")
+        elif gitfile == "ui/webui/webui-desktop":
```
Comment (KKoukiou) [inline scripts/makeupdates:279]: I think this commit contradicts with what https://github.com/rhinstaller/anaconda/commit/d7c9c7aca5b5af804b51e187510eeaae6ffec47d introduced a few months ago. 

Please sync with @M4rtinK

Comment (jkonecny12) [reply scripts/makeupdates:279]: I don't think it does. This is just about the startup script not Javascript files which were issue with the commit.

Comment (M4rtinK) [reply scripts/makeupdates:279]: @jkonecny12 is correct in this - the change i did was to not copy the overall Web UI files, as we now have logic in place that auto-starts the Web UI if the `anaconda-webui` package is installed: https://github.com/rhinstaller/anaconda/pull/4871/files and this just wont work right now, without a custom kickstart file pointing to a tar payload on the boot.iso.

As long as makeupdates does not drag in `/usr/share/cockpit/anaconda-webui` then the change in this PR is fine.

Still, even https://github.com/rhinstaller/anaconda/commit/d7c9c7aca5b5af804b51e187510eeaae6ffec47d is just a band-aid - ideally we need both a saner logic for Web UI startup as well as being able to actually run the Web UI with real world payloads on the boot.iso - which effectively means finally adding support for RPMs & all the related UI - source selection, software selection, DNF transaction monitoring and error handling UI.

**Review by M4rtinK — APPROVED** [review]
Looks like this should work.

**Pull request merged**